.eslintrc

```diff
@@ -32,5 +32,13 @@
     "import/no-unresolved": "error",
     "import/no-unused-modules": ["off", { "unusedExports": true }],
     "no-redeclare": "off"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["src/devtools/**/*.{ts,tsx}"],
+      "rules": {
+        "import/no-unresolved": ["error", { "ignore": ["react-query"] }]
```

Comment:
NIT: Since we added `react-query` as an external and global for all entry points, I think it makes sense to configure this ignore to be global instead of a devtools-specific override as well? I definitely don't have strong feelings on it though.

Reply:
I just thought that maybe it's  a good idea to avoid wrong auto-imports or so, but being in line with the rollup config is also valuable: https://github.com/tannerlinsley/react-query/pull/2113/commits/7398b24819eaf263aae42f2eeaaa2aebe16ec8b8

uff, and the missing comma 🤦 
https://github.com/tannerlinsley/react-query/pull/2113/commits/1e18f1cab0bb9945ef9ddbd069f8d75bfb689a78